docs/reporting-apis.md

```diff
@@ -1,25 +1,25 @@
 # Renovate EE Reporting APIs
 
 Reporting APIs provide real-time data about the state of Orgs, Repos, and Pull requests that are managed by Mend Renovate.
-These APIs are available only on Mend Renovate Enterprise Edition instances running with a Postgres DB.
 
-**Version**: This documentation reflects APIs available in Mend Renovate Enterprise v7.0.0
+**Version**: This documentation reflects APIs available in Mend Renovate Enterprise v7.2.0
```

Comment:
Remove this. Docs are implicitly for the latest version anyway

Reply:
Done